### ISSUE TITLE
include hue manager for hadoop stack

### ISSUE THREAD

**Issue by lavvy** [rancher/compose-templates#4]
Please could you please find time to include a gui manager for hadoop . Some users are not techies : ) .

**Comment by cloudnautique** [comment]
@lavvy, will look into it. Will be a little bit before I get a chance to add something. Thanks for the feedback!

**Comment by cloudnautique** [comment]
@lavvy, it looks like Hue wants Oozie/Pig available to submit jobs via.. Yarn? I'm not 100% on the ecosystem, but does that sound right? I was hoping to keep this somewhat core with HDFS/Yarn, but can certainly see the utility of HUE.

**Comment by lavvy** [comment]
Yeah,  it helps most hadoop apps,  i think we should have some relevant
hadoop apps too like impala for sql like operations.
I can't remember again where i came across a docker image that includes
most of the relevant apps including impala ,  maybe i should look for it
again if it's necessary
